### ISSUE TITLE
Pre-trained Model not working

### ISSUE THREAD

**Issue by frankwen-jwah** [pawn-sxy/DCAMA#7]
![image](https://user-images.githubusercontent.com/45443590/212942526-1de84617-d2a8-4364-b532-6a56e874b0b5.png)
Is the pre-trained model provided ready for use? I got all random predictions in outputs.

 Or do we have to train from scratch? Thank you.

**Comment by pawn-sxy** [comment]
The pre-trained model is ready for use. The calculation of mIoU is the average of mean IoU of each class in the test set and the number of class is set fixed in our code, so if your samples did not cover all the class the results should not be right. You can refer the 'compute_iou()' in ./commom/logger.py.
